sonar-bungee/src/main/java/xyz/jonesdev/sonar/bungee/fallback/FallbackChannelHandler.java

```diff
@@ -40,12 +41,13 @@ public FallbackChannelHandler(final @NotNull Channel channel) {
     super(channel);
   }
 
-  private static final Field CHANNEL_WRAPPER_FIELD;
+  private static final MethodHandle CHANNEL_WRAPPER_GETTER;
 
   static {
     try {
-      CHANNEL_WRAPPER_FIELD = HandlerBoss.class.getDeclaredField("channel");
-      CHANNEL_WRAPPER_FIELD.setAccessible(true);
+      CHANNEL_WRAPPER_GETTER = MethodHandles
+        .privateLookupIn(HandlerBoss.class, MethodHandles.lookup())
```

Comment:
Please use the same codestyle as here: <https://github.com/jonesdevelopment/sonar/blob/main/sonar-velocity/src/main/java/xyz/jonesdev/sonar/velocity/fallback/FallbackChannelInitializer.java#L38>